### ISSUE TITLE
Importing pygwalker affects logging in Jupyter Notebook

### ISSUE THREAD

**Issue by Jython1415** [Kanaries/pygwalker#178]
## Description

I found that importing pygwalker in my notebook causes all logging messages to be printed to a cell's output. Importing other packages (such as numpy or pandas) does not cause this to happen, so I thought it ought to be reported here. If this could be changed (or a workaround suggested) that would be great.

## Reproducible example

Run examples Jupyter Notebook. I have only tested this on my local machine, so if someone could confirm whether this is reproducible or not, that would be great.

### Does not print (other imports)

```Python
import logging
import numpy
import pandas

logger = logging.getLogger()
logger.setLevel(logging.DEBUG)
logger.debug("test")
```

### Does print (pygwalker import)

```Python
import logging
import pygwalker

logger = logging.getLogger()
logger.setLevel(logging.DEBUG)
logger.debug("test")
```

**Comment by Jython1415** [comment]
The source of the issue:

https://github.com/Kanaries/pygwalker/blob/8c981cc38ccf5068001a85b816a05a6b4c0d6ad6/pygwalker/__init__.py#L15

Is this line necessary? I am not sure if it is common practice for packages to use `basicConfig` as this can impact externally configured logger. However, I am new to this so please let me know what makes sense. I would be happy to help.

**Comment by longxiaofei** [comment]
good feedback! it will be fixed.

**Comment by Jython1415** [comment]
@longxiaofei thank you! And thank you for the tool. It is incredibly convenient.

**Comment by longxiaofei** [comment]
fixed in version 0.2.0,
